server/mailet/mock-smtp-server/pom.xml

```diff
@@ -119,4 +119,84 @@
             <version>3.1.7</version>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>jib-maven-plugin</artifactId>
+                <version>1.5.0</version>
+                <configuration>
+                    <from>
+                        <image>openjdk:8u222-jre</image>
+                    </from>
+                    <to>
+                        <image>linagora/mock-smtp-server</image>
+                        <tags>
+                            <tag>0.1</tag>
```

Comment:
you probably don't want to push a maven SNAPSHOT version to a docker tag, don't you?